src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -187,7 +187,6 @@ public override void Start()
 
             this.disposableResources.Add(this.timeSyncBehaviorState);
             this.disposableResources.Add(this.chainRepository);
-            this.disposableResources.Add(this.connectionManager);
```

Comment:
Where is `LoggerFactory` getting disposed?
Can you also do the same with `LoggerFactory` remove it from this list and put it where ever we dispose LoggerFactory